### PR TITLE
List voice messages method

### PR DESCRIPTION
**Pull request by Faultless**
This PR adds:

- [X] List Voice messages method
- [X] Example & Test cases

### CODE REVIEW

File: Examples/VoiceMessage/ListVoiceMessages.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{
+    class ListVoiceMessages
+    {
+        const string YourAccessKey = "YOUR_ACCESS_KEY"; // your access key here.
+
+        static void Main(string[] args)
+        {
+            IProxyConfigurationInjector proxyConfigurationInjector = null; // for no web proxies, or web proxies not requiring authentication
+            //proxyConfigurationInjector = new InjectDefaultCredentialsForProxiedUris(); // for NTLM based web proxies
+            //proxyConfigurationInjector = new InjectCredentialsForProxiedUris(new NetworkCredential("domain\\user", "password")); // for username/password based web proxies
```
Comment (danielsequeira) [inline Examples/VoiceMessage/ListVoiceMessages.cs]: why the commented code?

Comment (Faultless) [reply Examples/VoiceMessage/ListVoiceMessages.cs]: removed

**Comment by Faultless** [comment]
There's still an issue with Referencing `MessageBird` in the Test Project, which I am not experienced enough to debug. My changes don't appear to me to possibly cause such a build issue.
It clearly shows here that the reference is in: https://github.com/messagebird/csharp-rest-api/blob/master/Tests/UnitTests/MessageBirdUnitTests/MessageBirdUnitTests.csproj#L7-L9

**Comment by danielsequeira** [comment]
@Faultless aren't you missing an import in your test class?
`Resources/VoiceMessageTest.cs(19,26): error CS0103: The name 'Client'`
don't you get this error locally?

**Comment by Faultless** [comment]
@danielsequeira  yes, that was it! thanks

**Pull request merged**